app/auth/__init__.py

```
@@ -11,26 +11,17 @@ async def verify_token(request: Request) -> bool:
     bearer_prefix = "Bearer "
 
     if not auth_header or not auth_header.startswith(bearer_prefix):
-        raise HTTPException(
-            status_code=status.HTTP_401_UNAUTHORIZED,
-            detail="Invalid or missing token",
-            headers={"WWW-Authenticate": "Bearer"},
-        )
+        return False  # Invalid or missing token
 
     token = auth_header[len(bearer_prefix) :]
     headers = {"Authorization": f"Bearer {token}"}
-    print(VERIFICATION_URL)
-    print(token)
+
     async with httpx.AsyncClient() as client:
         response = await client.get(VERIFICATION_URL, headers=headers)
 
     if response.status_code == 200:
         json_response = response.json()
         if "id" in json_response:
-            return token
+            return True  # Token is valid
 
-    raise HTTPException(
-        status_code=status.HTTP_401_UNAUTHORIZED,
-        detail="Invalid token",
-        headers={"WWW-Authenticate": "Bearer"},
-    )
+    return False  # Invalid token
```

i want to raise an error. maybe we could add it as part of logging? not sure.